### PR TITLE
doc: ESS kit update24.8

### PR DESCRIPTION
**Pull request by birgitZF**
## Description

Update documentation for ESS kit for release 24.8
see also https://github.com/eclipse-tractusx/sig-release/issues/638

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/ESS-Kit/page_adoption-view.md [hunk 1]
```diff
@@ -88,7 +88,14 @@ The trace of an incident follows previously mentioned principles along any oblig
 
 ![ESS_KitCustomerJourney](assets/CatenaX_ESS_KitCustomerJourney.svg)
 
-**Note:** There will be an explanatory youtube video, which will be available after IP clearance. [ESS video](<https://bcgcatenax.sharepoint.com/:v:/r/sites/ARTV/Shared%20Documents/ESS%20(Environment%20and%20Social%20Standards)/ESS%20KIT/Film/CX-full%20process%20demo%20v5.mp4?csf=1&web=1&e=nVeYZ6>)
+**Note:** There is an explanatory video published on youtube. [ESS video](<https://youtu.be/ySsy6v9_bsM>)
+
+### Customer Journey: Occasion-related tracking in broken supply chains
+
+If no Catena-X member can be identified within the Catena-X network, other external sources could be used to identify the missing suppliers and to fill the gaps that are relevant for this ESS incident.
```
Comment (stephanbcbauer) [inline docs-kits/kits/ESS-Kit/page_adoption-view.md:95]:
```suggestion
If no Catena-X member can be identified within the Catena-X network, other external sources could be used to identify the missing suppliers and fill the gaps that are relevant for this ESS incident.
```

File: docs-kits/kits/ESS-Kit/page_adoption-view.md [hunk 2]
```diff
@@ -88,7 +88,14 @@ The trace of an incident follows previously mentioned principles along any oblig
...
+
+For each supplier that could be found, the Clearing Agency would need to check against the Catena-X network, if a BPN is available or not. The identified supplier (there could be more than one), who is a member of the Catena-X network, will be assigned as ESS incident moderator for this ESS incident. The incident moderator can be addressed via the EDC endpoint of the Business Partner and is most likely the incident manager of this Business Partner.
```
Comment (stephanbcbauer) [inline docs-kits/kits/ESS-Kit/page_adoption-view.md:97]:
```suggestion
For each supplier that could be found, the Clearing Agency would need to check against the Catena-X network if a BPN is available or not. The identified supplier (there could be more than one), who is a member of the Catena-X network, will be assigned as the ESS incident moderator for this ESS incident. The incident moderator can be addressed via the EDC endpoint of the Business Partner and is most likely the incident manager of this Business Partner.
```

File: docs-kits/kits/ESS-Kit/page_adoption-view.md [hunk 3]
```diff
@@ -88,7 +88,14 @@ The trace of an incident follows previously mentioned principles along any oblig
...
+Now the tracing of an ESS incident can continue as described above. The ESS incident moderator shall trace the supply chain both upstream (towards his suppliers) and downstream (towards his customers). For Business Partners, that are members of the supply chain, the Catena-X network and its services can be utilized, for Business Partners, that are not in the Catena-X network other means of communication must be used.
```
Comment (stephanbcbauer) [inline docs-kits/kits/ESS-Kit/page_adoption-view.md:98]:
```suggestion
Now, the tracing of an ESS incident can continue as described above. The ESS incident moderator shall trace the supply chain both upstream (towards his suppliers) and downstream (towards his customers). For Business Partners, that are members of the supply chain, the Catena-X network and its services can be utilized, for Business Partners, that are not in the Catena-X network, other means of communication must be used.
```

File: docs-kits/kits/ESS-Kit/page_adoption-view.md [hunk 4]
```diff
@@ -174,7 +181,7 @@ The data model follows the Catena-X Standard [CX-0144](<https://catena-x.net/de/
 
 ### Access & Usage Policies / Code of Conduct
 
-#### Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards
+Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards
```
Comment (stephanbcbauer) [inline docs-kits/kits/ESS-Kit/page_adoption-view.md:184]: From my perspective, the #### was correct? Isn't it a header?

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
LGTM, some minor grammar changes and a question

File: docs-kits/kits/ESS-Kit/page_adoption-view.md (hunk 1 above)
Comment (birgitZF) [reply docs-kits/kits/ESS-Kit/page_adoption-view.md:95]: fixed

File: docs-kits/kits/ESS-Kit/page_adoption-view.md (hunk 2 above)
Comment (birgitZF) [reply docs-kits/kits/ESS-Kit/page_adoption-view.md:97]: fixed

File: docs-kits/kits/ESS-Kit/page_adoption-view.md (hunk 3 above)
Comment (birgitZF) [reply docs-kits/kits/ESS-Kit/page_adoption-view.md:98]: fixed

File: docs-kits/kits/ESS-Kit/page_adoption-view.md (hunk 4 above)
Comment (birgitZF) [reply docs-kits/kits/ESS-Kit/page_adoption-view.md:184]: instead of just bold line, use header level 4 (####)

**Review by danielmiehle — APPROVED** [review]
LGTM

**Review by jSchuetz88 — APPROVED** [review]
Looks good to me.

But wasn't there an intended update regarding the Code of Conduct?

**Comment by birgitZF** [comment]
> Looks good to me.
> 
> But wasn't there an intended update regarding the Code of Conduct?

@jSchuetz88 , yes we wanted to add a link to the Code of Conduct. As we do not know where the Code of Coduct will be published, we cannot refer to that location. Unless somebody can give us this information ;)

**Comment by stanfaldin** [comment]
> > Looks good to me.
> > But wasn't there an intended update regarding the Code of Conduct?
> 
> @jSchuetz88 , yes we wanted to add a link to the Code of Conduct. As we do not know where the Code of Coduct will be published, we cannot refer to that location. Unless somebody can give us this information ;)

HI @jSchuetz88  we talked about it last week, you wanted to create a dummy file or sth like that on the CX site until the final CoC file is released through the opt out phase in the "non-technical standards" area.

**Comment by jSchuetz88** [comment]
> > > Looks good to me.
> > > But wasn't there an intended update regarding the Code of Conduct?
> > 
> > 
> > @jSchuetz88 , yes we wanted to add a link to the Code of Conduct. As we do not know where the Code of Coduct will be published, we cannot refer to that location. Unless somebody can give us this information ;)
> 
> HI @jSchuetz88 we talked about it last week, you wanted to create a dummy file or sth like that on the CX site until the final CoC file is released through the opt out phase in the "non-technical standards" area.

Unfortunatelly I wait for a PR on my own to add the dummy files, but the link will be: 
- for the english version: https://catenax-ev.github.io/docs/next/non-functional-requirements/assets/CX-NFR-ESS-Codex_v.1.0_EN.pdf
- for the german version https://catenax-ev.github.io/docs/next/non-functional-requirements/assets/CX-NFR-ESS-Codex_v.1.0_DE.pdf

File: docs-kits/kits/ESS-Kit/page_adoption-view.md [hunk 5]
```diff
@@ -184,7 +184,7 @@ The data model follows the Catena-X Standard [CX-0144](<https://catena-x.net/de/
 #### Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards
 
 In the Catena-X use case sustainability / environmental and social standards, principles are established with the aim of establishing transparent and trustworthy practices within the Catena-X network.
-[The Code of Conduct will be published on the Catena-X webpage](<https://catenax-ev.github.io/docs/next/non-functional-requirements/assets/CX-NFR-ESS-Codex_v.1.0_EN.pdf>).
+The [Code of Conduct will be published on the Catena-X webpage](<https://catenax-ev.github.io/docs/next/non-functional-requirements/assets/CX-NFR-ESS-Codex_v.1.0_EN.pdf>)
```
Comment (jSchuetz88) [inline docs-kits/kits/ESS-Kit/page_adoption-view.md:187]:
```suggestion
The [Code of Conduct will be published on the Catena-X webpage](https://catenax-ev.github.io/docs/next/non-functional-requirements/assets/CX-NFR-ESS-Codex_v.1.0_EN.pdf)
```

This should be rendered correctly by docusaurus after the merge.

**Review by maximilianong — APPROVED** [review]

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Comment by stanfaldin** [comment]
@stephanbcbauer or @maximilianong or @matbmoser can you merge it please

**Pull request merged**